docs/wikidata.md

```diff
@@ -0,0 +1,92 @@
+# Wikidata Full Quickstart
+
+The following instructions take you through loading the full Wikidata Knowledge
+Base in QLever. Since Wikidata is very large at over 7 billion triples, **we
+recommend to run all of the following commands in a `tmux` or `screen` session**.
+Nevertheless we will note below when a command will take long to execute.
+
+## Download and build QLever using `docker`
+
+We build QLever using `docker` which takes care of all necessary dependencies.
+This should only take a couple of minutes
+
+    git clone --recursive https://github.com/ad-freiburg/QLever.git qlever
+    cd qlever
+    docker build -t qlever .
+
+## Download Wikidata
+
+If you already downloaded **and decrompressed** Wikidata to uncompressed Turtle
+format you can skip this step, otherwise we download and uncompress it.
+
+**Note:** This takes several hours as Wikidata is about 42 GB compressed and the
+servers are throttled. Also make sure you have enough space on the filesystem
+where you unpacked QLever or see the full
```

Comment:
`unpacked` -> `unpack` (we haven't unpacked QLever yet)

Reply:
Good point, I was thinking about downloading a `tar` of QLever itself even though the instructions do a `git clone`